### PR TITLE
Milestone3 - Newsweek

### PR DESCRIPTION
**Pull request by Anvi98**
In the third and last milestone of the project, we completed the last sections of the website and prepared it for our portfolio.

Main sections ( US; World; Business; Tech & Science; Culture; Sports; Expert forum)

We Created these sections of the website for the 3 screen sizes.

Thank you

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1448 @@
...
+            <div class="category-label">
+              <a class="text-decoration-none" href="/world"> POLITICS </a>
+            </div>
+            <a href="#">
+              <img class="mb-2 w-100" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711224/israel-tanks-border-syria-golan.webp?w=397&h=265&f=7808a219e546114ab901f153186bd843" alt="cover" />
+            </a>
+          </div>
+
+          <h6 class="font-weight-bold cat-a mid-title">
+            Israel Security Officials Reveal Top Threats Faced on Four Fronts This Year
```
Comment (bini-i) [inline index.html]: Please provide links to the titles of each category (for this and all similar sections below :point_down: )

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1448 @@
...
+            <h6 class="font-weight-bold mid-title">
+              <a class="cat-a" href="#">Jared Padalecki on how his 'Walker' Reboot Differs From the
+              Original</a
+            >
+          </h6>
+        </div>
+
+        <div class="col-md-4">
+          <div class="d-inline-flex flex-nowrap border-bottom pb-1">
+            <img
```
Comment (bini-i) [inline index.html:1325]: Please provide links for images (here and in all matching sections below)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1448 @@
...
+              class="mr-2 w-75 align-self-center"
+              src="img/tiger-wild-animal-sanctuary-colorado.webp"
+              alt=""
+            />
+            <div>
+              <h6 class="font-weight-bold small-text mid-title">
+                'Tiger King' Zoo Owners Ordered To Hand Over Big Cats to
```
Comment (bini-i) [inline index.html:1332]: Please provide links for titles (here and in all matching sections below)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1448 @@
...
+                  <div class="details">With its pro-business agenda and open-border policy for investors, the Central European nation is hoping to bounce back from the COVID-19 crisis, while making greater strides toward innovation, research and development.</div>
+                </div>
+              </div>
+              <div class="news">
+                <form action="index.html" class="sign-up">
+                  <label for="email-address" class="label_title">
+                    <span>N</span>
+                    Start your day with our top 5 articles
+                  </label>
+                  <input type="text" id="email-address" class="form-text" placeholder="Email address">
```
Comment (bini-i) [inline index.html]: * [x] Please make sure Newsletter section has an input with type email

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes requred :yellow_circle:  
### Hello  team :wave: 

#### Very nice project, you have done really well precisely matching the source page plus you have a great README and commit history but you still can improve your project with the requirements suggested below:
- In the magazine, Editor's pick and main newspaper section, please make sure each title and image has a link and try giving a link to the image and title individually instead of wrapping both of them in a link tag. (refer to inline comments)
- Please make sure the Newsletter section has an input with type **email**
- Subscribe and newsletter section should match the source page on mobile view (image should be stacked on top of title) and fix the overflow
> Original 
image
> Clone 
image

#### You still have requirements to work on from milestone 2
- The layout below header should match the original page as 
* [x] you should have 2 column layout on medium screen (you currently have 3 column layout)
* [x] and on mobile screen you should match the following order for the sections
> image

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] You haven't done any commit after the previous code review, please make the changes asked by them and then ask for another.

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @Anvi98 :wave: 

Go on with the next task.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request closed without merging**